junit/mockwebserver/src/test/groovy/io/fabric8/mockwebserver/DefaultMockServerAnyMethodTest.groovy

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.fabric8.mockwebserver
+
+import io.vertx.core.Future
+import io.vertx.core.Vertx
+import io.vertx.ext.web.client.WebClient
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.util.concurrent.AsyncConditions
+
+class DefaultMockServerAnyMethodTest extends Specification {
+
+  @Shared
+  static def vertx = Vertx.vertx()
+
+  DefaultMockServer server
+
+  WebClient client
+
+  def setup() {
+    server = new DefaultMockServer()
+    server.start()
+    client = WebClient.create(vertx)
+  }
+
+  def cleanup() {
+    server.shutdown()
+    client.close()
+  }
+
+  def cleanupSpec() {
+    vertx.close()
+  }
+
+  def "when setting an expectation for ANY should respond to POST"() {
+    given: "An expectation with once"
+      server.expect().any().withPath("/api/v1/resource").andReturn(200, "OK").always()
```

Comment:
Umm, I might be wrong here but this statement doesn't seem to match what the description says. Should it be using `once()` instead?

Reply:
will update after merging https://github.com/fabric8io/kubernetes-client/pull/5671